hr_evaluation_360/models/hr_evaluation_plan_phase.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Luis Felipe Mileo - <mileo@kmee.com.br> - KMEE
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+"""
+    In human resources or industrial psychology, 360-degree feedback,
+    also known as multi-rater feedback, multi source feedback, or
+    multi source assessment, is a process utilized by organizations
+    to solicit information from a variety of workplace sources on
+    an employee's work-related behavior and/or performance.
+
+    Most often, information solicited in a 360-degree feedback process
+    will include feedback from an employee's subordinates, peers
+    (colleagues), and supervisor(s), as well as a self-evaluation by
+    the employee him or herself. Such feedback can also include, when
+    relevant, feedback from external sources who interact with the
+    employee, such as customers and suppliers or other interested
+    stakeholders. 360-degree feedback is so named because it solicits
+    feedback regarding an employee's behavior from a variety of points
+     of view (subordinate, lateral, and supervisory). It therefore may
+    be contrasted with "downward feedback" (traditional feedback on
+    work behavior and performance delivered to subordinates by supervisory
+    or management employees only; see traditional performance appraisal),
+    or "upward feedback" delivered to supervisory or management employees
+    by subordinates only.
+
+    Source: https://en.wikipedia.org/wiki/360-degree_feedback
+
+"""
+
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+from dateutil import parser
+import time
+
+from openerp import api, fields, models
+
+
+class HrEvaluationPlanPhase(models.Model):
+
+    _inherit = 'hr_evaluation.plan.phase'
+
+    action = fields.Selection(
+        selection_add=[
+            ('360', u'360-degree feedback'),
+        ]
+    )
+
+
+class HrEvaluationEvaluation(models.Model):
+
+    _inherit = 'hr_evaluation.evaluation'
+
+    @api.multi
+    def button_plan_in_progress(self):
+        hr_eval_inter_obj = self.env['hr.evaluation.interview']
+        for evaluation in self:
+            wait = False
+            for phase in evaluation.plan_id.phase_ids:
+                if phase.action != '360':
+                    return super(HrEvaluationEvaluation,
+                                 self).button_plan_in_progress()
```

Comment:
Yes because core implementation set state as waiting withouy any validation 

https://github.com/odoo/odoo/blob/8.0/addons/hr_evaluation/hr_evaluation.py#L230